packages/expo-task-manager/src/TaskManager.ts

```diff
@@ -133,9 +199,3 @@ if (ExpoTaskManager) {
     }
   );
 }
-
-// @tsapeta: Turn off `defineTask` function right after the initialization phase.
```

Comment:
Are we planning to replace this mechanism w some other one?

Reply:
I don't see any other way to do what we want here and that works well with fast refresh, so I just removed that and allowed users to redefine their tasks.